python/ray/__init__.py

```diff
@@ -60,12 +60,17 @@
     ActorClassID,
     ActorID,
     ClientID,
+    Count,
     Config as _Config,
+    Histogram,
+    Gauge,
     JobID,
     WorkerID,
     FunctionID,
     ObjectID,
     ObjectRef,
+    Sum,
```

Comment:
Could we avoid exporting these functions in the root APIs? It is reasonable to have `ray.ObjectRef` or `ray.Language`, but `ray.Sum` & `ray.Count` seems quite strange. We can create a new python module called `metrics` and put the definitions under it.

Reply:
It makes sense. I'll pack them into a new metric module.